ui/raidboss/data/03-hw/raid/a2s.js

```diff
@@ -90,18 +90,10 @@
     {
       'locale': 'fr',
       'replaceSync': {
-        'Added new combatant Boomtype Magitek Gobwalker G-VII': 'Ajout d\'un nouveau combattant Gobblindé magitek G-VII Boumbardier',
-        'Added new combatant Giant Bomb': 'Ajout d\'un nouveau combattant Bombe géante',
-        'Added new combatant Gordian Hardhelm': 'Ajout d\'un nouveau combattant Casque-dur gordien',
-        'Added new combatant Gordian Hardmind': 'Ajout d\'un nouveau combattant Cerveau-dur gordien',
-        'Added new combatant Gordian Sniper G-IX': 'Ajout d\'un nouveau combattant Sniper gordien',
-        'Added new combatant Jagd Doll': 'Ajout d\'un nouveau combattant Poupée jagd',
-        'Added new combatant Magitek Gobwidow G-IX': 'Ajout d\'un nouveau combattant Gobmygale magitek G-IX',
-        'Bomb': 'Bombe',
         'Boomtype Magitek Gobwalker G-VII': 'Gobblindé magitek G-VII Boumbardier',
-        'Hangar 12 will be sealed off in 15 seconds': 'Fermeture du grand hangar GH-12 dans 15 secondes.',
```

Comment:
For the zone sealing message, you can make this:
`'Hangar 12': 'grand hangar GH-12',`

This is because https://github.com/quisquous/cactbot/blob/master/ui/raidboss/common_replacement.js#L5-L10 automatically takes care of the Fermeture etc part.